WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -281,30 +281,23 @@ private async Task<bool> InitialiseTransactionAsync()
 
 			return await Task.Run(() => TransactionHelpers.BuildTransaction(_wallet, _info, tryToSign: false));
 		}
-		catch (NotEnoughFundsException ex)
+		catch (Exception ex) when (ex is NotEnoughFundsException or TransactionFeeOverpaymentException || (ex is InvalidTxException itx && itx.Errors.OfType<FeeTooHighPolicyError>().Any()))
```

Comment:
Can the `(ex is InvalidTxException itx && itx.Errors.OfType<FeeTooHighPolicyError>().Any())` line be shorter?

Reply:
I don't think so, at least I didn't find a way to do so.